### ISSUE TITLE
add cli option to enable/disable redshift

### ISSUE THREAD

**Issue by aster94** [jonls/redshift#604]
hello Jon,

i think that an option to enable/diable redshift from the command line would be useful, right now it is possible only from the redshift-gtk with a user button

**Comment by jonls** [comment]
What is the right way of implementing this? Maybe when running in continuous mode Redshift should write out the process ID to a file so a signal can be sent to the process.

I think I'd probably prefer this to be in a separate executable (e.g. `redshift-control`) because there's enough confusion about the options of current `redshift` executable as it is.

**Comment by MichaelMackus** [comment]
`killall redshift` works fine for me 🤷

**Comment by stpeters00** [comment]
Even better is `killall -s USR1 redshift`, which can be the only command in a `toggle-redshift` script.

**Comment by ngdio** [comment]
Depending on your OS, redshift comes with an systemd user service. That way you can start redshift with `systemctl --user start redshift` and stop it with `systemctl --user stop redshift`. Wouldn't work with the GTK applet though.

**Comment by stpeters00** [comment]
Fedora 28, which I'm using, comes with a systemd user redshift service.  The redshift service file `/usr/lib/systemd/user/redshift.service` has, in the [Unit] section:

`After=display-manager.service`

I don't use any desktop system - I roll my own with raw X and fvwm, so I don't have any
display-manager service.  Does the redshift service really need one?

**Comment by ngdio** [comment]
No, just means the display manager would be started before redshift, but only if you have one installed. Otherwise it'd be `Required`.
